applications/CompressiblePotentialFlowApplication/tests/naca0012_small_transonic_test/naca0012_small_transonic_parameters.json

```diff
@@ -21,6 +21,14 @@
         "echo_level"             : 0,
         "relative_tolerance": 1e-12,
         "absolute_tolerance": 1e-12,
+        "transonic_scheme_settings"          : {
+            "initial_critical_mach"          : 0.9,
+            "initial_upwind_factor_constant" : 1.0,
+            "target_critical_mach"           : 0.9,
+            "target_upwind_factor_constant"  : 1.0,
+            "update_relative_residual_norm"  : 1e-3,
+            "mach_number_squared_limit"      : 3.0
```

Comment:
Wouldn't be more user-friendly to ask for `limit_mach_number` and do the square inside?

Reply:
Yes, it's a bit strange but that's what it's called in literature. Until now, I always used it set to 3 and I haven't changed it in all the cases I tried. I'm not sure about changing the name just because we might lose the reference.

Reply:
If so, I'd leave it as it is customary in the literature :+1: .